aries-site/src/pages/components/skeleton-loading.mdx

```diff
@@ -0,0 +1,33 @@
+import { Notification } from 'grommet';
+import { Example } from '../../layouts';
+import { DashBoardSkeletonExample, SkeletonAnatomy } from '../../examples';
+
+<Example
+  componentName="Skeleton"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/skeleton/DashBoardSkeletonExample.js"
+  docs="https://v2.grommet.io/skeleton?theme=hpe#props"
+>
+  <DashBoardSkeletonExample />
+</Example>
+
+### When to use
+Skeletons are used when:
+- Loading elements are known ahead of time but are not immediately available.
+- All of the content on a page is loaded in at the same time.
+- Users are loading high-traffic pages, such as a user dashboard.
```

Comment:
Wouldn't skeletons apply to any page? not just high-traffic ones

Reply:
@KennyAtHPE tagging for eyes